WSLDVCPlugin/WSLDVCCallback.cpp

```diff
@@ -605,10 +613,12 @@ class WSLDVCCallback :
         }
 
         // build wsl.exe/wslg.exe command line.
-        // -d : distro.
-        // --cd : current directory.
-        // -- : executable path with arguments.
-        if ((swprintf_s(exeArgs, ARRAYSIZE(exeArgs), L"-d %s --cd %s -- %s",
+        // -d : distro name, space is not allowed in distro name,
+        //      thus wrapping by double-quoto is not accepted by wsl.exe/wslg.exe
+        // --cd : current directory, wrap by double-quote.
+        // -- : executable path with arguments. All string after '--' will be treated as Linux command line,
+        //      thus wrapping by double-quoto is not accepted by wsl.exe/wslg.exe
+        if ((swprintf_s(exeArgs, ARRAYSIZE(exeArgs), L"-d %s --cd \"%s\" -- %s",

```

Comment:
Thanks, fixed. This is a bit tricky since double quote (") is valid file/path char in Linux (while it is not in Windows), so it's possible Linux working folder may contain double quote, but by following up with owner of wsl.exe/wslg.exe, currently it can't handle it, so until it gets fixed or decided on how to pass such as arguments, here block path contains double quote.